### PR TITLE
DMU: Golden Argosy + 5 cards

### PR DESCRIPTION
**Pull request by Suthro**
DMU: Add scripts for 6 new DMU cards

- Archangel of Wrath
- Aron, Benalia's Ruin
- Garna, Bloodfist of Keld
- Golden Argosy
- Rona, Sheoldred's Faithful
- Rona's Vortex

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/archangel_of_wrath.txt [hunk 1]
```diff
@@ -0,0 +1,12 @@
+Name:Archangel of Wrath
+ManaCost:2 W W
+Types:Creature Angel
+PT:3/4
+K:Kicker:B:R
+K:Flying
+K:Lifelink
+T:Mode$ ChangesZone | ValidCard$ Card.Self+kicked | Origin$ Any | Destination$ Battlefield | Execute$ TrigDamage | TriggerDescription$ When CARDNAME enters the battlefield, if it was kicked, it deals 2 damage to any target.
+T:Mode$ ChangesZone | ValidCard$ Card.Self+kicked | Origin$ Any | Destination$ Battlefield | CheckSVar$ X | SVarCompare$ EQ2 | Execute$ TrigDamage | TriggerDescription$ When CARDNAME enters the battlefield, if it was kicked twice, it deals 2 damage to any target.
```
Comment (Hanmac) [inline forge-gui/res/cardsfolder/upcoming/archangel_of_wrath.txt:9]: for this i asked the rules manager about "kicked twice" or stuff that might increase the Kicker amount like "Sticker Kicker" but they didn't answer me yet

File: forge-gui/res/cardsfolder/upcoming/golden_argosy.txt [hunk 2]
```diff
@@ -0,0 +1,15 @@
+Name:Golden Argosy
+ManaCost:4
+Types:Artifact Vehicle
+PT:3/6
+T:Mode$ BecomesCrewed | ValidVehicle$ Card.Self | Execute$ RememberCrew | Static$ True
+SVar:RememberCrew:DB$ PumpAll | ValidCards$ Creature.TriggeredCrew | RememberAllPumped$ True
+T:Mode$ Attacks | ValidCard$ Card.Self | TriggerZones$ Battlefield | Execute$ TrigFlicker | TriggerDescription$ Whenever CARDNAME attacks, exile each creature that crewed it this turn. Return them to the battlefield tapped under their owner’s control at the beginning of the next end step.
+SVar:TrigFlicker:DB$ ChangeZoneAll | ChangeType$ Creature.IsRemembered | Mandatory$ True | Origin$ Battlefield | Destination$ Exile | SubAbility$ DelTrig
```
Comment (Hanmac) [inline forge-gui/res/cardsfolder/upcoming/golden_argosy.txt:8]: there should be a "Crewed this Turn" Property, so you don't need RemeberCrew

File: forge-gui/res/cardsfolder/upcoming/rona_sheoldreds_faithful.txt [hunk 3]
```diff
@@ -0,0 +1,11 @@
+Name:Rona, Sheoldred's Faithful
+ManaCost:1 U B B
+Types:Legendary Creature Human Wizard
+PT:3/4
+T:Mode$ SpellCast | ValidCard$ Instant,Sorcery | ValidActivatingPlayer$ You | TriggerZones$ Battlefield | Execute$ TrigDrain | TriggerDescription$ Whenever you cast or copy an instant or sorcery spell, each opponent loses 1 life.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/rona_sheoldreds_faithful.txt:5]: ~~or copy~~

Comment (Suthro) [reply forge-gui/res/cardsfolder/upcoming/rona_sheoldreds_faithful.txt]: Fixed

File: forge-gui/res/cardsfolder/upcoming/archangel_of_wrath.txt (hunk 1 above)
Comment (Suthro) [reply forge-gui/res/cardsfolder/upcoming/archangel_of_wrath.txt:9]: Okay, I'll wait on an update for now. I can update this script in the future and revisit Temporal Firestorm in the same way if necessary.

File: forge-gui/res/cardsfolder/upcoming/golden_argosy.txt (hunk 2 above)
Comment (Suthro) [reply forge-gui/res/cardsfolder/upcoming/golden_argosy.txt:8]: Sorry, are you able to elaborate more? I couldn't find a `CrewedThisTurn` property across current card scripts. Is this what you say you are fixing?

**Review by tool4ever — APPROVED** [review]

File: forge-gui/res/cardsfolder/upcoming/archangel_of_wrath.txt [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+SVar:TrigDamage:DB$ DealDamage | ValidTgts$ Creature,Player,Planeswalker | TgtPrompt$ Select any target | NumDmg$ 2
+SVar:X:Count$TimesKicked
+Oracle:\nKicker {B} and/or {R} (You may pay an additional {B} and/or {R} as you cast this spell.)\nFlying, lifelink\nWhen Archangel of Wrath enters the battlefield, if it was kicked, it deals 2 damage to any target.\nWhen Archangel of Wrath enters the battlefield, if it was kicked twice, it deals 2 damage to any target.
```
Comment (paulsnoops) [inline forge-gui/res/cardsfolder/upcoming/archangel_of_wrath.txt:12]: Oracle:\nKicker

\n isn't needed here

File: forge-gui/res/cardsfolder/upcoming/golden_argosy.txt [hunk 5]
```diff
@@ -0,0 +1,15 @@
+Name:Golden Argosy
+ManaCost:4
+Types:Artifact Vehicle
+PT:3/6
+T:Mode$ BecomesCrewed | ValidVehicle$ Card.Self | Execute$ RememberCrew | Static$ True
+SVar:RememberCrew:DB$ PumpAll | ValidCards$ Creature.TriggeredCrew | RememberAllPumped$ True
+T:Mode$ Attacks | ValidCard$ Card.Self | TriggerZones$ Battlefield | Execute$ TrigFlicker | TriggerDescription$ Whenever CARDNAME attacks, exile each creature that crewed it this turn. Return them to the battlefield tapped under their owner’s control at the beginning of the next end step.
```
Comment (paulsnoops) [inline forge-gui/res/cardsfolder/upcoming/golden_argosy.txt:7]: owner’s = owner's

File: forge-gui/res/cardsfolder/upcoming/ronas_vortex.txt [hunk 6]
```diff
@@ -0,0 +1,8 @@
+Name:Rona's Vortex
+ManaCost:U
+Types:Instant
+K:Kicker:2 B
+A:SP$ ChangeZone | ValidTgts$ Creature.YouDontCtrl,Planeswalker.YouDontCtrl | TgtPrompt$ Select target creature or planeswalker | Origin$ Battlefield | Destination$ Hand | SubAbility$ DBChangeZone | ConditionCheckSVar$ X | ConditionSVarCompare$ EQ0 | SpellDescription$ Return target creature or planeswalker you don't control to its owner's hand. If this spell was kicked, put that permanent on the bottom of its owner's library instead.
+SVar:DBChangeZone:DB$ ChangeZone | Defined$ Targeted | Origin$ Battlefield | Destination$ Library | LibraryPosition$ -1 | Condition$ Kicked
+SVar:X:Count$TimesKicked
+Oracle:\nKicker {2}{B} (You may pay an additional {2}{B} as you cast this spell.)\nReturn target creature or planeswalker you don't control to its owner's hand. If this spell was kicked, put that permanent on the bottom of its owner's library instead.
```
Comment (paulsnoops) [inline forge-gui/res/cardsfolder/upcoming/ronas_vortex.txt:8]: Oracle:\nKicker

\n isn't needed here

File: forge-gui/res/cardsfolder/upcoming/golden_argosy.txt [hunk 7]
```diff
@@ -0,0 +1,15 @@
...
+SVar:TrigFlicker:DB$ ChangeZoneAll | ChangeType$ Creature.IsRemembered | Mandatory$ True | Origin$ Battlefield | Destination$ Exile | SubAbility$ DelTrig
+SVar:DelTrig:DB$ DelayedTrigger | Mode$ Phase | Phase$ End of Turn | Execute$ TrigReturn | RememberObjects$ RememberedLKI | SubAbility$ DBCleanup | TriggerDescription$ Return them to the battlefield tapped under their owner's control at the beginning of the next end step.
+SVar:TrigReturn:DB$ ChangeZone | Origin$ Exile | Destination$ Battlefield | Defined$ DelayTriggerRemembered | Tapped$ True
+T:Mode$ Phase | Phase$ End of Turn | Static$ True | Execute$ DBCleanup
+SVar:DBCleanup:DB$ Cleanup | ClearRemembered$ True
+K:Crew:1
+SVar:HasAttackEffect:True
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/golden_argosy.txt:14]: `True` > `TRUE`

**Pull request merged**